client/src/pages/Contest/NewContestForm/NewContestForm.tsx

```diff
@@ -0,0 +1,220 @@
+                error={touched.title && Boolean(errors.title)}
+                value={values.title}
+                onChange={handleChange}
+              />
+            </Grid>
+            <Grid item xs={11} md={9}>
+              <label htmlFor="description">Description</label>
+              <br />
+              <TextField
+                id="description"
+                multiline
+                variant="outlined"
+                fullWidth
+                margin="none"
+                name="description"
+                placeholder="Details about what type of tattoo you want"
+                rows={4}
+                helperText={touched.description ? errors.description : ''}
+                error={touched.description && Boolean(errors.description)}
+                value={values.description}
+                onChange={handleChange}
+              />
+            </Grid>
+            <Grid item xs={11} md={9}>
+              <Grid container spacing={4}>
+                <Grid item xs={12} md={4}>
+                  <label htmlFor="prizeAmount">Prize amount</label>
+                  <br />
+                  <OutlinedInput
+                    id="prizeAmount"
+                    fullWidth
+                    type="number"
+                    inputProps={{
+                      min: '1',
+                      max: '1000000',
+                    }}
+                    margin="none"
+                    name="prizeAmount"
+                    placeholder="100.00"
+                    startAdornment={<InputAdornment position="start">$</InputAdornment>}
+                    error={touched.prizeAmount && Boolean(errors.prizeAmount)}
+                    value={values.prizeAmount}
+                    onChange={handleChange}
+                  />
+                </Grid>
+                <Grid item xs={12} md={8}>
+                  <label>Deadline</label>
+                  <Grid container spacing={4}>
+                    <Grid item xs={12} md={6}>
+                      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <KeyboardDatePicker
+                          disableToolbar
+                          variant="inline"
+                          format="dd MMM yyyy"
+                          margin="none"
+                          id="date"
+                          name="date"
+                          fullWidth
+                          inputVariant="outlined"
+                          value={selectedDate}
+                          KeyboardButtonProps={{
+                            'aria-label': 'change date',
+                          }}
+                          onChange={handleDateChange}
+                        />
+                      </MuiPickersUtilsProvider>
+                    </Grid>
+                    <Grid item xs={12} md={3}>
+                      <Select fullWidth id="time" name="time" variant="outlined" defaultValue="">
+                        {times.map((t) => (
```

Comment:
Maybe consider using a Time Picker, it will make things a lot easier for you :) 

Ref: https://material-ui.com/components/pickers/